### ISSUE TITLE
No option for setting the height of images

### ISSUE THREAD

**Issue by ManalLiaquat** [Expertizo/react-fb-image-grid#2]
There is not an option (props) to set the height of images, when I am using this library in Material UI CARD component it takes the 100% height and cannot be set by custom styling (CSS).
@kashifsulaiman 

![download](https://user-images.githubusercontent.com/36041100/46416793-b51a4700-c741-11e8-9749-8283d6b71874.png)
First one is without the card and the second one is inside the card component.

**Comment by kashifsulaiman** [comment]
@ManalLiaquat Thanks for reporting, will fix it soon!

**Comment by kashifsulaiman** [comment]
@ManalLiaquat We can't provide the height explicitly because we want the picture's height to be same as width to show them equal in height and width. This issue has been resolved in the version `0.1.3`, Checkout this PR https://github.com/Expertizo/react-fb-image-grid/pull/6.
Closing this issue. Feel free to reopen if you still find the issue in `0.1.3`

**Comment by ManalLiaquat** [comment]
> @ManalLiaquat We can't provide the height explicitly because we want the picture's height to be same as width to show them equal in height and width. This issue has been resolved in the version `0.1.3`, Checkout this PR #6.
> Closing this issue. Feel free to reopen if you still find the issue in `0.1.3`

@kashifsulaiman I am still having trouble in UI, using latest `version=0.1.3`. Here is the screenshot 
![screenshot_1](https://user-images.githubusercontent.com/36041100/46541828-723da800-c8d6-11e8-92bf-88d7a3edb56a.png)


and using like this: `<FacebookImage images={post.images} countFrom={5} width={35} fromRight={100} />` using `width={35}`  because it is not get set with the card at `width={100}`. I think it is due to that you are using `position: absolute` in the CSS file.

**Comment by kashifsulaiman** [comment]
@ManalLiaquat 
Yes this is because of position `absolute` because we don't want our images to be dependent on parent's height.
For your issue, Wrap the `FbImageLibrary` component with a div, provide it the same height as the same you provided `width` to the component. e.g.
```
        <div style={{height: '50vw'}}>
          <FbImageLibrary images={images} width={50}/>
        </div>

        <div style={{height: '80vw'}}>
          <FbImageLibrary images={images} width={80}/>
        </div>
```

**Comment by ManalLiaquat** [comment]
### I uses this technique also, but did'nt work 🤔

> @ManalLiaquat
> Yes this is because of position `absolute` because we don't want our images to be dependent on parent's height.
> For your issue, Wrap the `FbImageLibrary` component with a div, provide it the same height as the same you provided `width` to the component. e.g.
> 
> ```
>         <div style={{height: '50vw'}}>
>           <FbImageLibrary images={images} width={50}/>
>         </div>
> 
>         <div style={{height: '80vw'}}>
>           <FbImageLibrary images={images} width={80}/>
>         </div>
> ```


@kashifsulaiman I use this technique but didn't work for both devices (PC and mobile).

**CODE:**
`<div style={{ height: "34vw" }}><FacebookImage images={post.images} width={34} align="left" /></div>`

**PC Screenshot:**

![download](https://user-images.githubusercontent.com/36041100/46573605-2a3f8380-c9b1-11e8-9041-4d5390902243.png)

**Mobile Screenshot:**

![download 1](https://user-images.githubusercontent.com/36041100/46573611-36c3dc00-c9b1-11e8-8c83-962040ac17c1.png)

Now the mobile view is getting unpleasant. 🙄

**Comment by kashifsulaiman** [comment]
Yes this is because 'vw' is 'view-width'. You've set '34vw' for the web, not for mobile. So for mobile, put another value. You've to make the conditional expression.
Like wise, detect if mobile then '100vw' else '34vw'.

**Comment by ManalLiaquat** [comment]
> Yes this is because 'vw' is 'view-width'. You've set '37vw' for the web, not for mobile. So for mobile, put another value. You've to make the conditional expression.
> Like wise, detect if mobile then '100vw' else '37vw'.

I am also thinking about this conditional expression but I did'nt do that because this Library is using react_bootstrap so it should be responsive. 😉🤗

**Comment by kashifsulaiman** [comment]
@ManalLiaquat which library are you using for these cards? Let me know so I could test on these!

**Comment by kashifsulaiman** [comment]
@ManalLiaquat Fixed these bugs in `v0.1.4`
PR: https://github.com/Expertizo/react-fb-image-grid/pull/8

**Comment by ManalLiaquat** [comment]
> @ManalLiaquat which library are you using for these cards? Let me know so I could test on these!

Material-UI

**Comment by ManalLiaquat** [comment]
> @ManalLiaquat Fixed these bugs in `v0.1.4`
> PR: #8

Thank you to fix this issue, this library is working now correctly on both; mobile and desktop devices.
